lib/binTree/parsingTree.c

```diff
@@ -0,0 +1,171 @@
+#include "parsingTree.h"
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+
+struct Node
+{
+    char* value;
+    Node* leftChild;
+    Node* rightChild;
+};
+
+Node* buildTree(char expression[100][100], int* pointer)
+{
+    char* item = expression[*pointer];
+    ++(*pointer);
+    if (!item[0])
+    {
+        return NULL;
+    }
+    if (strstr("+-/*", item))
+    {
+        Node* operator = calloc(1, sizeof(Node));
+        if (operator == NULL)
+        {
+            return NULL;
+        }
+        operator->value = calloc(2, sizeof(char));
+        if (operator->value == NULL)
+        {
+            return NULL;
+        }
+        strcpy(operator->value, item);
+
+        Node* operand1 = calloc(1, sizeof(Node*));
```

Comment:
Если мы тут выделяем память под `Node*`, то возвратит calloc `Node**`, тут типы неправильные. Наверное, имелось в виду
```suggestion
        Node* operand1 = calloc(1, sizeof(Node));
```